layouts/default.vue

```diff
@@ -120,42 +182,93 @@
 	import { readNotifications, updateNotifications } from '@directus/sdk';
 	import { defaults } from 'chart.js';
 	import capitalize from 'lodash/capitalize';
+	import { useInboxNotificationIds } from '~/composables/useInboxNotificationIds';
 	import { useAuth } from '~/store/auth';
+	import { formatDateTime } from '~/utils/date-formatters';
 
 	const { $directus } = useNuxtApp();
 
 	const auth = useAuth();
 	const user = auth.getUser as User;
 
 	// NOTIFICATIONS
+	const notificationBus = useEventBus<string[]>('notification-updated');
+
+	notificationBus.on((ids) => {
```

Comment:
Please rename `ids` to be more descriptive, e.g. `idsToArchive` or whatever else it represents.